### PR TITLE
feat: Run linter before compilation

### PR DESCRIPTION
**Pull request by ryanbtp**
* Fix the lower case role errors it found

Dependent on a .4 release of Chronicle

Signed-off-by: Ryan Roberts <ryan@blockchaintp.com>

### CODE REVIEW

**Comment by scealiontach** [comment]
Hold this until 0.4.0 release

**Review by scealiontach — CHANGES_REQUESTED** [review]
hold until 0.4

File: domains/evidence/domain.yaml [hunk 1]
```diff
@@ -12,6 +12,8 @@ attributes:
     type: String
   Version:
     type: Int
+  Versionies:
```
Comment (scealiontach) [inline domains/evidence/domain.yaml:15]: What's this? Looks like a typo

**Review by scealiontach — CHANGES_REQUESTED** [review]

File: domains/evidence/domain.yaml (hunk 1 above)
Comment (suchapalaver) [reply domains/evidence/domain.yaml]: Corrected.

**Review by scealiontach — APPROVED** [review]

**Pull request merged**